ui/hooks/useFiatFormatter.ts

```diff
@@ -1,35 +1,89 @@
 import { useSelector } from 'react-redux';
 import { getIntlLocale } from '../ducks/locale/locale';
 import { getCurrentCurrency } from '../selectors';
+import { shortenString } from '../helpers/utils/util';
 
 /**
  * Returns a function that formats a fiat amount as a localized string.
+ * The hook takes an optional boolean parameter to shorten the fiat value.
  *
  * Example usage:
  *
  * ```
  * const formatFiat = useFiatFormatter();
  * const formattedAmount = formatFiat(1000);
+ *
+ * const shorteningFiatFormatter = useFiatFormatter(true);
+ * const shortenedAmount = shorteningFiatFormatter(100000000000000000);
  * ```
  *
  * @returns A function that takes a fiat amount as a number and returns a formatted string.
  */
 
+const TRUNCATED_CHAR_LIMIT_FOR_SHORTENED_FIAT = 15;
+const TRUNCATED_START_CHARS_FOR_SHORTENED_FIAT = 12;
+
 type FiatFormatter = (fiatAmount: number) => string;
 
-export const useFiatFormatter = (): FiatFormatter => {
+export const useFiatFormatter = (shortenFiatValue?: boolean): FiatFormatter => {
```

Comment:
Since this is a boolean, should we use an object argument to make it more readable?